src/test/scala/tech/cryptonomic/conseil/tezos/DataTypesTest.scala

```diff
@@ -4,20 +4,41 @@ import java.sql.Timestamp
 
 import org.scalamock.scalatest.MockFactory
 import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.{Matchers, WordSpec}
+import org.scalatest.{BeforeAndAfterEach, Matchers, WordSpec}
+import tech.cryptonomic.conseil.config.Platforms
+import tech.cryptonomic.conseil.config.Platforms.{PlatformsConfiguration, TezosConfiguration, TezosNodeConfiguration}
 import tech.cryptonomic.conseil.generic.chain.DataTypes._
 import tech.cryptonomic.conseil.generic.chain.PlatformDiscoveryTypes.{Attribute, DataType, Entity, KeyType}
-import tech.cryptonomic.conseil.metadata.{EntityPath, MetadataService, NetworkPath, PlatformPath}
+import tech.cryptonomic.conseil.metadata._
 
-import scala.concurrent.{ExecutionContext, Future}
-
-class DataTypesTest extends WordSpec with Matchers with ScalaFutures with MockFactory {
+class DataTypesTest extends WordSpec with Matchers with ScalaFutures with MockFactory with BeforeAndAfterEach {
   import scala.concurrent.ExecutionContext.Implicits.global
 
-  val ms = stub[MetadataService]
-  val testEntityPath = EntityPath("testEntity", NetworkPath("testNetwork", PlatformPath("testPlatform")))
+  val platformDiscoveryOperations = new TestPlatformDiscoveryOperations
+  val cacheOverrides = stub[AttributeValuesCacheConfiguration]
+
+  def createMetadataService(stubbing: => Unit = ()): MetadataService = {
+    stubbing
+
+    new MetadataService(
+      PlatformsConfiguration(
+        Map(
+          Platforms.Tezos -> List(
+                TezosConfiguration("alphanet", TezosNodeConfiguration("tezos-host", 123, "https://"))
+              )
+        )
+      ),
+      new TransparentUnitTransformation(),
+      cacheOverrides,
+      platformDiscoveryOperations
+    )
+  }
+
+  val testEntityPath = EntityPath("testEntity", NetworkPath("alphanet", PlatformPath("tezos")))
   val testEntity = Entity("testEntity", "Test Entity", 0)
 
+  override protected def beforeEach(): Unit = platformDiscoveryOperations.clear()
```

Comment:
as mentioned above this can be exchanged with mixing in trait `OneInstancePerTest`, also why in this case there is no `super.beforeEach?`

Reply:
done